sdk/apis/apis/v1alpha2/types_apibinding_test.go

```diff
@@ -105,3 +105,110 @@ func TestAPIBindingPermissionClaimCELValidation(t *testing.T) {
 		})
 	}
 }
+
+// TestAPIBindingAPIExportReferenceCELValidation will validate the APIExport reference for an otherwise valid APIBinding.
+func TestAPIBindingPermissionClaimSelectorCELValidation(t *testing.T) {
+	testCases := []struct {
+		name         string
+		current, old map[string]any
+		wantErrs     []string
+	}{
+		{
+			name: "no change",
+			current: map[string]any{
+				"matchAll": true,
+			},
+			old: map[string]any{
+				"matchAll": true,
+			},
+		},
+		{
+			name:    "nothing set",
+			current: map[string]any{},
+			old:     map[string]any{},
+			wantErrs: []string{
+				"openAPIV3Schema.properties.spec.properties.permissionClaims.items.properties.selector: Invalid value: \"object\": a selector is required. Only \"matchAll\" is currently implemented",
+			},
```

Comment:
Just for the record, I hate asserting exact error strings.

Reply:
ACK!